src/libraries/System.Management/src/System.Management.csproj

```diff
@@ -9,6 +9,9 @@
     <Nullable>annotations</Nullable>
     <IncludeDllSafeSearchPathAttribute>true</IncludeDllSafeSearchPathAttribute>
     <IsPackable>true</IsPackable>
+    <!-- If you enable GeneratePackageOnBuild for this package and bump ServicingVersion, make sure to also bump ServicingVersion in Microsoft.Windows.Compatibility.csproj once for the next release. -->
+    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+    <ServicingVersion>1</ServicingVersion>
```

Comment:
Proactively avoiding the merge conflict with https://github.com/dotnet/runtime/pull/83549/files by bumping the version to 1.

Reply:
Meh, didn't work. GitHub considered it a conflict as soon as I merged the other PR. Oh well.